install/Knative-with-GKE.md

```diff
@@ -135,32 +135,56 @@ rerun the command to see the current status.
 > Note: Instead of rerunning the command, you can add `--watch` to the above
   command to view the component's status updates in real time. Use CTRL + C to exit watch mode.
 
-## Installing Knative Serving
+## Installing Knative components
 
-1. Next, we will install [Knative Serving](https://github.com/knative/serving)
-and its dependencies:
+You have the option to install and use only the Knative components that you
+want. You can install only the component of Knative if you need that
+functionality, for example Knative serving is not required to create and run
+builds.
+
+### Installing Knative Serving
```

Comment:
Installing serving also installs build, so it might be confusing to users who think they have to do both.

Reply:
Sam and I discussed changing the language to "serving and build" or "build only" to clarify.